js/__fixtures__/credentials/bbs/deriveProofFrame.json

```diff
@@ -0,0 +1,14 @@
+{
+    "@context": [
+      "https://www.w3.org/2018/credentials/v2",
```

Comment:
same here and probably many others too I bet, gonna just say find and replace these and not mention the other places.

Reply:
same as before, until there is a real URL, no reason to change this.